lib/bloc/user_profile/user_profile_bloc.dart

```diff
@@ -151,30 +151,41 @@ class UserProfileBloc {
             color: randomName[0],
             animal: randomName[1]);
       }
-
       user = user.copyWith(locked: user.securityModel.requiresPin);
+      await _checkBiometrics().then(
```

Comment:
If using await then it is better to do:
`enrolledBiometrics  = await _checkBiometrics();`

Reply:
[Done](https://github.com/breez/breezmobile/pull/225/commits/deda31cf880894cfd5741e2c9cb8cd0b11fdc52f)